packages/server/src/server.js

```diff
@@ -43,9 +43,12 @@ async function createApp(options) {
   // While LHCI should be served behind nginx/apache that handles compression, it won't always be.
   app.use(compression());
 
-  // 1. Support large payloads because LHRs are big.
-  // 2. Support JSON primitives because `PUT /builds/<id>/lifecycle "sealed"`
-  app.use(bodyParser.json({limit: '10mb', strict: false}));
+  if (typeof useBodyParser === 'undefined' || useBodyParser) {
```

Comment:
little late but.. maybe we flip the flag so we don't have to do this kinda check?

`dontUseBodyParser` or `skipBodyParser` ?